torch/distributed/distributed_c10d.py

```diff
@@ -718,6 +718,8 @@ def irecv(tensor,
     if group == GroupMember.WORLD:
         _check_default_pg()
         return _default_pg.recv([tensor], src, tag)
+    elif src is None:
```

Comment:
Can we update the docs for `src` to indicate this behavior (similar to `recv`) and also add a default `src=None` in the function?

Reply:
Updated `irecv` definition to default src=None, and changed docs to match `recv`